microshift_release_notes/microshift-4-15-release-notes.adoc

```diff
@@ -229,4 +229,18 @@ Adds OLM image references in dedicated file::
 
 Issued: 2024-03-20
 
-{product-title} release 4.15.3 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:1257[RHBA-2024:1257] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:1255[RHSA-2024:1255] advisory.
\ No newline at end of file
+{product-title} release 4.15.3 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:1257[RHBA-2024:1257] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:1255[RHSA-2024:1255] advisory.
+
+[id="microshift-4-15-4-dp"]
+=== RHBA-2024:1451 - {microshift-short} 4.15.4 bug fix and enhancement update advisory
+
+Issued: 2024-03-26
+
+{product-title} release 4.15.4 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:1451[RHBA-2024:1451] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:1449[RHSA-2024:1449] advisory.
+
+[discrete]
+[id="microshift-4-15-4-enhancements"]
+==== Enhancements
+
+Change in OpenShift-marketplace namespace security::
```

Comment:
@aravipra 
`OpenShift-marketplace` to `openshift-marketplace`

Reply:
@aravipra you change OpenShift-marketplace to openShift-marketplace, but it should be openshift-marketplace (all are low cases)

please correct it, thanks

Reply:
done